internal.nix

```diff
@@ -139,17 +265,15 @@ rec {
               )}
 
             if grep -I -q -r '/bin/' .; then
-              source $TMP/preinstall-env
+              source $stdenv/setup
```

Comment:
Why did you revert this change? This was consciously done to speed up the repetitive sourcing. IIRC it provided a >2x build speed improvement (multiple minutes to sub one minute) on big projects.

Reply:
uh.. that was totally not intentional. I think i made a mistake during a rebase! I'll fix it